### ISSUE TITLE
Datagrip/Java crash at launch

### ISSUE THREAD

**Issue by DaveSpe** [flathub/com.jetbrains.DataGrip#91]
After updating to latest version 2023.1 commit d30e082d099c12ecc25bae3fbad5752f6a229172, java/datagrip is crashing.
Was working fine prior to the update.

![image](https://user-images.githubusercontent.com/65930418/229178546-b0ea94e1-1cd2-482f-9fe0-ca09e362f348.png)
First I get error: Cannot connect to already running IDE instance.
CannotActivateException: Process 2 is still running

Then when I relaunch I get a brief glimpse of the datagrip interface and crash to desktop.
![image](https://user-images.githubusercontent.com/65930418/229177571-918dfa04-8712-4929-bfd1-0626d4ae396e.png)
![image](https://user-images.githubusercontent.com/65930418/229177865-d4044637-4247-4d85-9fac-b97524092b27.png)

**Comment by DaveSpe** [comment]
> Mine too, I see this when I try to run with `flatpak run`:
> 
> ```
> [0331/173232.716386:FATAL:setuid_sandbox_host.cc(157)] The SUID sandbox helper binary was found, but is not configured correctly. Rather than run without sandboxing I'm aborting now. You need to make sure that /app/extra/datagrip/jbr/lib/chrome-sandbox is owned by root and has mode 4755.
> ```

Good call didn't think to run it via command line. Get the same as you when I do.

**Comment by wolfymaster** [comment]
I know nothing about how this is packaged and how to solve the issue, but what I can say is that if I go here:
```~/.local/share/flatpak/app/com.jetbrains.DataGrip/current/active/files/extra/datagrip/bin```
and run `./datagrip.sh` then it _does_ start up correctly and is functional. 

But trying to run it via `flatpack run com.jetbrains.DataGrip` fails as you've documented above.

**Comment by FakeShemp** [comment]
The new Chromium sandbox needs zypak. See here for fix: https://github.com/flathub/com.jetbrains.PyCharm-Community/pull/534

**Comment by x80486** [comment]
Closed via #92

**Comment by Snowbridge** [comment]
same here. A couple days ago DataGrip has updated and is unable to start since then
```console
lms@lms-ubuntu:~$ head -n 4 /etc/os-release
PRETTY_NAME="Ubuntu 22.04.3 LTS"
NAME="Ubuntu"
VERSION_ID="22.04"
VERSION="22.04.3 LTS (Jammy Jellyfish)"
lms@lms-ubuntu:~$ uname -a
Linux lms-ubuntu 6.2.0-26-generic #26~22.04.1-Ubuntu SMP PREEMPT_DYNAMIC Thu Jul 13 16:27:29 UTC 2 x86_64 x86_64 x86_64 GNU/Linux
lms@lms-ubuntu:~$ flatpak list --app | grep Grip
DataGrip	com.jetbrains.DataGrip	2023.2.1	stable	system
```
running from terminal:
```console
lms@lms-ubuntu:~$ flatpak run com.jetbrains.DataGrip
[0.001s][warning][perf,memops] Cannot use file /tmp/hsperfdata_lms/2 because it is locked by another process (errno = 11)
CompileCommand: exclude com/intellij/openapi/vfs/impl/FilePartNodeRoot.trieDescend bool exclude = true

Start Failed
Cannot connect to already running IDE instance.
Exception: Process 2 is still running
WARN: <resource-bundle>messages.DataGripBundle</resource-bundle> tag is found in an xml descriptor included into the platform part of the IDE but the platform part uses predefined bundles (e.g. ActionsBundle for actions) anyway; this tag must be replaced by a corresponding attribute in some inner tags (e.g. by 'resource-bundle' attribute in 'actions' tag)
Gtk-Message: 09:35:18.266: Failed to load module "canberra-gtk-module"
Gtk-Message: 09:35:18.266: Failed to load module "canberra-gtk-module"
```
and there is no `/tmp/hsperfdata_lms` directory. Tried to create `/tmp/hsperfdata_lms/2`, but made no difference: the error message is the same and the directory `/tmp/hsperfdata_lms` vanished after `flatpak run`.

Solved by downgrading to 2023.1.2 (tried also previous 2023.2, but it's broken either):

```console
lms@lms-ubuntu:~$ sudo flatpak update --commit=65cdd322fd9fffbad5b0a0108225c4593aa85ccf49ec9ae11640e21caf87ad4c com.jetbrains.DataGrip
```
